zuul.d/arista-eos-jobs.yaml

```diff
@@ -49,117 +30,74 @@
-    parent: network-ee-integration-tests-arista-eos
-    vars:
       test_ansible_network_cli_ssh_type: libssh
-      test_fips_mode: true
-
-# =============================================================================
-- job:
-    name: network-ee-integration-tests-arista-eos-stable-2.12
-    parent: ansible-network-eos-appliance
-
-- job:
-    name: network-ee-integration-tests-arista-eos-stable-2.12
-    parent: network-ee-integration-tests-stable-2.12
+    host-vars:
+      eos-4.24.6:
+        ansible_network_os: eos
     nodeset: eos-4.24.6-python38
-    vars:
-      test_ansible_python_interpreter: /usr/bin/python3.8
-      test_ansible_ssh_private_key_file: ~/.ssh/id_ecdsa
-      test_ansible_skip_tags: httpapi
-      # Backport PR to be merged https://github.com/ansible/ansible/pull/75706 to enable fips for libssh job
-      test_fips_mode: false
+    timeout: 10800
 
 - job:
-    name: network-ee-integration-tests-arista-eos-httpapi-stable-2.12
-    parent: network-ee-integration-tests-arista-eos-stable-2.12
+    name: ansible-ee-integration-arista-eos-httpapi-stable-2.9
+    parent: ansible-ee-integration-arista-eos-httpapi-latest
     vars:
-      test_ansible_skip_tags: network_cli
+      ansible_runner_container_version: stable-2.9-devel
 
 - job:
-    name: network-ee-integration-tests-arista-eos-libssh-stable-2.12
-    parent: network-ee-integration-tests-arista-eos-stable-2.12
+    name: ansible-ee-integration-arista-eos-httpapi-stable-2.11
+    parent: ansible-ee-integration-arista-eos-httpapi-latest
     vars:
-      test_ansible_network_cli_ssh_type: libssh
-      test_fips_mode: true
+      ansible_runner_container_version: stable-2.11-devel
 
-# =============================================================================
 - job:
-    name: network-ee-integration-tests-arista-eos-stable-2.11
-    parent: ansible-network-eos-appliance
-
-- job:
-    name: network-ee-integration-tests-arista-eos-stable-2.11
-    parent: network-ee-integration-tests-stable-2.11
-    nodeset: eos-4.24.6-python38
+    name: ansible-ee-integration-arista-eos-httpapi-stable-2.12
+    parent: ansible-ee-integration-arista-eos-httpapi-latest
     vars:
-      test_ansible_python_interpreter: /usr/bin/python3.8
-      test_ansible_ssh_private_key_file: ~/.ssh/id_ecdsa
-      test_ansible_skip_tags: httpapi
-      # Backport PR to be merged https://github.com/ansible/ansible/pull/75706 to enable fips for libssh job
-      test_fips_mode: false
-
-- job:
-    name: network-ee-integration-tests-arista-eos-httpapi-stable-2.11
-    parent: network-ee-integration-tests-arista-eos-stable-2.11
-    vars:
-      test_ansible_skip_tags: network_cli
+      ansible_runner_container_version: stable-2.12-devel
 
 - job:
-    name: network-ee-integration-tests-arista-eos-libssh-stable-2.11
-    parent: network-ee-integration-tests-arista-eos-stable-2.11
+    name: ansible-ee-integration-arista-eos-httpapi-libssh-latest
```

Comment:
@goneri libssh is not a httpapi connection. We need to remove httpapi from the job name.